src/sonarwhal-theme/layout/partials/rules-index.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="layout layout--halves">
+    {{#each categories}}
+    <div class="module placement">
+        <p class="title icon--categories icon--a11y">{{name}}</p>
```

Comment:
I don't think the `icon--a11y` should be here. We need to find a way to change it depending on the category.

Reply:
I'm going to change the class name to match with the category name i.e: icon--a11y to icon--accessibility

With this change we can change the icon easily

Reply:
btw, there is no icon for `development`